django/thunderstore/security/utils.py

```diff
@@ -0,0 +1,140 @@
+import base64
+import binascii
+import json
+
+import requests
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives.asymmetric import ec
+from six import b, text_type
+
+
+class DERStringDecoder(object):
```

Comment:
`(object)` is no longer necessary in Python 3 (and in fact discouraged), you can use `class DERStringDecoder:`